bin/run

```diff
@@ -32,11 +32,11 @@ wait_for() {
 
 case $1 in
   web)
-    newrelic-admin run-python manage.py migrate
+    newrelic-admin run-python manage.py migrate --noinput
     exec newrelic-admin run-program gunicorn atmo.wsgi:application -b 0.0.0.0:${PORT} --workers 4 --access-logfile -
     ;;
   web-dev)
-    python manage.py migrate
+    python manage.py migrate --noinput
```

Comment:
I had a permission that was added in a different branch and not having this `--no-input` here basically broke the run of the web contains.